### PR TITLE
feat: phone app deployment docs

### PR DESCRIPTION
**Pull request by momentiris**
I started writing documentation/guide for how the deployment with fastlane works and how it's set up at the moment. Still a WIP so keep as draft until both me and Tom have finished it and looked at it again.

### CODE REVIEW

File: content/phone-app/_index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+```
+> We used the ssh url (git@...) the first time for creating the certificates etc. The one you see in the current match file is for bypassing ssh setup on CI. Now, instead, we use a github personal access token with repo privileges for authenticating. You can see how we use it in the top of the Matchfile.
+
+This will run match with parts of the configuration that is specified in your `Matchfile`. Since the repository will be empty at this stage, match will prompt for your consent to generate a new certificate and provisioning profile and put it there.
+
+It will also ask you to provide a password that can decrypt the repository data on subsequent operations. **This password is set as MATCH_PASSWORD environment variable in our CI pipeline (travis).**
+
+Using match for deployments:
+
+For this setup, we used a separate apple user that we setup and added to the apple dev team.
```
Comment (lessp) [inline content/phone-app/_index.md]:
```suggestion
For this setup, we used a separate Apple user that we setup and added to the Apple dev team.
```

File: content/phone-app/_index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+**The user email is declared as the environment variable FASTLANE_USER in our CI**
+
+**The user password is declared as the environment variable FASTLANE_PASSWORD in our CI**
+
+Fastlane looks for these variables automatically and uses them if specified.
+
+Now we take a look in the `ios/Fastfile`, and the lane `ios_testflight`: 
+
+`setup_travis` Don't know exactly what this does but it's needed.
```
Comment (lessp) [inline content/phone-app/_index.md]:
```suggestion
- [`setup_travis`](https://docs.fastlane.tools/actions/setup_travis/) configures the keychain and profiles for use with Travis.
```

File: content/phone-app/_index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+*We use [travis file encryption](https://docs.travis-ci.com/user/encrypting-files/) for handling all the relevant secrets for the android build and deploy step. In the `.travis.yml`s android `before_install` step, we decrypt those files that are available in the build context as secrets and we decrypt them and put them in the right directories. The files are the **google_play.json**, **release.keystore**, and the **gradle.properties**.*
+
+- Google dev console development team.
+- Secret JSON file with android specific signing details. Get from Google dev console.
+- `gradle.properties` file with app specific signing variables.
+- `release.keystore` file that also handles signing in some way.
+
+Going through the `android/fastlane/Fastfile`:
+
+`gradle(...)` Builds the app with gradle and tells it to make a `bundleRelease`, which generates a .aab bundle that is x64 and x86 compatible.
```
Comment (lessp) [inline content/phone-app/_index.md]:
```suggestion
`gradle(...)` Builds the app with gradle and tells it to make a `bundleRelease`, which generates an `.aab`-bundle that is x64 and x86 compatible.
```

**Review by lessp — APPROVED** [review]
Nice work! Left a few suggestions, otherwise LGTM! 💯

**Comment by ixuz** [comment]
The docs looks good 👍

**Review by ixuz — APPROVED** [review]

**Pull request merged**